src/ItemTemplates/PeekRelationshipTemplate/PeekRelationshipTemplate.vstemplate

```diff
@@ -0,0 +1,42 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<VSTemplate Version="3.0.0" Type="Item" xmlns="http://schemas.microsoft.com/developer/vstemplate/2005" xmlns:sdk="http://schemas.microsoft.com/developer/vstemplate-sdkextension/2010">
+  <TemplateData>
+    <Name>Peek Relationship</Name>
+    <Description>Add a new Peek Relationship to a Visual Studio Package</Description>
+    <Icon>PeekRelationshipIcon.png</Icon>
+    <TemplateID>VsixPeekRelationshipCSharp</TemplateID>
+    <AppliesTo>VSIX + CSharp</AppliesTo>
+    <ProjectType>CSharp</ProjectType>
+    <RequiredFrameworkVersion>4.5</RequiredFrameworkVersion>
+    <NumberOfParentCategoriesToRollUp>1</NumberOfParentCategoriesToRollUp>
+    <DefaultName>PeekRelationship.cs</DefaultName>
+  </TemplateData>
+  <TemplateContent>
+    <References>
+      <Reference><Assembly>Microsoft.CSharp</Assembly></Reference>
+      <Reference><Assembly>Microsoft.VisualStudio.CoreUtility</Assembly></Reference>
+      <Reference><Assembly>Microsoft.VisualStudio.Editor</Assembly></Reference>
+      <Reference><Assembly>Microsoft.VisualStudio.Language.Intellisense</Assembly></Reference>
+      <Reference><Assembly>Microsoft.VisualStudio.OLE.Interop</Assembly></Reference>
+      <Reference><Assembly>Microsoft.VisualStudio.TextManager.Interop</Assembly></Reference>
+      <Reference><Assembly>Microsoft.VisualStudio.TextManager.Interop.8.0</Assembly></Reference>
+      <Reference><Assembly>Microsoft.VisualStudio.Text.UI.Wpf</Assembly></Reference>
+      <Reference><Assembly>Microsoft.VisualStudio.Text.UI</Assembly></Reference>
+      <Reference><Assembly>Microsoft.VisualStudio.Text.Data</Assembly></Reference>
+      <Reference><Assembly>Microsoft.VisualStudio.Text.Logic</Assembly></Reference>
+      <Reference><Assembly>System</Assembly></Reference>
+      <Reference><Assembly>System.ComponentModel.Composition</Assembly></Reference>
+    </References>
+    <ProjectItem ReplaceParameters="true" TargetFileName="$rootSafeItemName$PeekableItem.cs" OpenInEditor="true">PeekableItem.cs</ProjectItem>
+    <ProjectItem ReplaceParameters="true" TargetFileName="$rootSafeItemName$ItemSourceProvider.cs">PeekableItemSourceProvider.cs</ProjectItem>
```

Comment:
given the list of files it looks like "Peek provider" can be a better name for this template